src/NzbDrone.Core/MediaFiles/MediaInfo/MediaInfoFormatter.cs

```diff
@@ -100,6 +100,11 @@ public static string FormatAudioCodec(MediaInfoModel mediaInfo, string sceneName
 
             if (audioFormat.ContainsIgnoreCase("E-AC-3"))
             {
+                 if (splitAdditionalFeatures.ContainsIgnoreCase("JOC"))
+                {
```

Comment:
Indention is funky on one of these lines (L103 or 104). Other than that it looks OK

Reply:
I'm not sure which indentation you meant, but the one which looked like it had an extra space I removed and now it does not pass the checks.

Reply:
Just ignore those two tests, they only fail because you are pr from an external branch. Should be good now